docs/Sources/tuist/tuist.docc/Articles/Tuist/Users/sharing-code-across-manifests.md

```diff
@@ -0,0 +1,85 @@
+
+# Sharing code across manifests
+
+Project description helpers are a group of Swift files that can be accessed from any project manifest. They are useful to extract common patterns, define project conventions, or simplify the definition of projects.
+
+## Overview
+
+One of the inconveniences of Xcode when we use it with large projects is that it doesn't allow reusing elements of the projects other than the build settings through `.xcconfig` files. Being able to reuse project definitions is useful for the following reasons:
+
+- It eases the **maintenance** because changes can be applied in one place and all the projects get the changes automatically.
+- It makes it possible to define **conventions** that new projects can conform to.
+- Projects are more **consistent** and therefore the likelihood of broken builds due inconsistencies is significantly less.
+- Adding a new projects becomes an easy task because we can reuse the existing logic.
+
+Reusing code across manifest files is possible in Tuist thanks to the concept of **project description helpers**.
+
+### Definition
+
+Project description helpers are Swift files that get compiled into a framework, `ProjectDescriptionHelpers`, that manifest files can import.
+
+> Structure:
+Tuist is not opinionated about the files structure and the content in them, so it's up to you to give them a structure that makes sense for your project.
```

Comment:
The documentation hasn't mentioned yet where they are located. I'd mention that first: "Tuist considers all the sources of the `ProjectDescriptionHelpers` module all the files under `Tuist/ProjectDescriptionHelpers`". I would then remove the comment about being or not opinionated because that's captured by the "all the files in the directory (and sub-directories)"